harness/determined/exec/prep_container.py

```diff
@@ -241,6 +241,18 @@ def do_proxy(sess: Session, allocation_id: str) -> None:
         info = det.ClusterInfo._from_env()
         info._to_file()
 
+    try:
+        # See the ClusterInfo.trial property for explanation
+        debug = info.trial._debug
+    except (AssertionError, RuntimeError):
+        debug = False
```

Comment:
how about:
```python
debug = info.task_type == "TRIAL" and info.trial._debug
```
?

It's shorter, and follows the usage pattern described in the ClusterInfo docstring.

Reply:
Because that still raises an exception! You can see here: http://34.168.20.150:8080/det/experiments/15/logs